src/features/colinks/CoLinksContext.tsx

```diff
@@ -64,26 +64,28 @@ const CoLinksProvider: React.FC<CoLinksProviderProps> = ({ children }) => {
   // }, [onCorrectChain]);
 
   useEffect(() => {
-    if (data) {
-      if (data?.profile) {
-        if (!data.profile.invite_code_redeemed_at) {
-          navigate(coLinksPaths.wizard);
-        } else if (!data.profile.tos_agreed_at) {
-          navigate(coLinksPaths.wizard);
-        } else if (!data.profile.cosoul) {
-          // show the mint button
-          navigate(coLinksPaths.wizard);
-        } else if (!data.profile.links_held) {
-          // redirect to wizard so they can buy their own link
-          // we might already be on the wizard
-          if (location.pathname !== coLinksPaths.wizard) {
+    if (address) {
```

Comment:
This change is the hairiest to me. I also don't see this working - If I visit /home it takes me to /start

Reply:
i.e. there is no way to proceed without connecting wallet

Reply:
i figured out i can test with /explore